pyqt-configure

```diff
@@ -1,18 +1,25 @@
 #!/bin/sh
 
 
-PYTHON=python2
+PYTHON=python3
 PYTHON_VERSION=$(${PYTHON} -c 'import sys; print(sys.version[:3])')
 
-
 ${PYTHON} configure.py \
+    --assume-shared \
     --confirm-license \
     --enable=QtCore \
     --enable=QtGui \
     --enable=QtNetwork \
+    --enable=QtWidgets \
     --no-designer-plugin \
     --no-docstrings \
+    --no-qml-plugin \
     --no-qsci-api \
+    --sysroot=/app \
```

Comment:
When I was testing I didn't need this option. Did you find it to be necessary for something I missed?

Reply:
Aha!

So with `--sysroot=/app` we can remove `--bindir`, `--destdir` and `--sipdir`!

How did I miss this option in the first place, it's been there since 2014. :open_mouth: